router/eventorder_test.go

```diff
@@ -144,8 +144,10 @@ func TestEventOrderGuarantee(t *testing.T) {
 			// statusInsertLoop
 			config.Set("Router.updateStatusBatchSize", 100)
 			config.Set("Router.maxStatusUpdateWait", "10ms")
-
-			defer config.Reset()
+			config.Set("SourceDebugger.origin", "event_order_source_"+trand.String(100))
+			config.Set("DestinationDebugger.origin", "event_order_destination_"+trand.String(100))
+			config.Set("TransformerDebugger.origin", "event_order_transformer_"+trand.String(100))
+			config.Set("LiveEvent.cache.GCTime", "1s")
```

Comment:
are these really needed?
```suggestion
			defer config.Reset()
```

Reply:
Why do need two `defer config.Reset()`? We already have one at the start of the function right

Reply:
The tests are passing for me even without these four `config.Set` lines. The `config.Reset()` and `defer config.Reset()` on top seem to work.